index.php

```diff
@@ -1,650 +1,660 @@
-<?php
```

Comment:
Not sure why u have changed spacings. Maybe it was someone else's code structure that u have included in ur commit.